### PR TITLE
Update validators.bg.xlf

### PR DESCRIPTION
**Pull request by LubenZA**
Translated 83, 84, & 85

| Q             | A
| ------------- | ---
| Branch?       | master for features / 3.4 up to 4.2 for bug fixes 
| Bug fix?      | yes
| New feature?  | no 
| BC breaks?    | no     
| Deprecations? | no 
| Tests pass?   | yes    
| Fixed tickets | #30151
| License       | MIT
| Doc PR        | symfony/symfony-docs#...

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.bg.xlf [hunk 1]
```diff
@@ -318,6 +318,18 @@
...
+                <source>This is not a valid UUID.</source>
+                <target>Това не е валиден UUID.</target>
+            </trans-unit>
+            <trans-unit id="84">
+                <source>This value should be a multiple of {{ compared_value }}.</source>
+                <target>Тази стойност трябва да бъде кратно число на {{ compared_value }}.</target>
+            </trans-unit>
+            <trans-unit id="85">
+                <source>This Business Identifier Code (BIC) is not associated with IBAN {{ iban }}.</source>
+                <target>Tози бизнес идентификационен код (BIC) не е свързана с IBAN {{ iban }}.</target>
```
Comment (javiereguiluz) [inline src/Symfony/Component/Validator/Resources/translations/validators.bg.xlf]: Google Translate tells me that `Tози` should be `Този`. There's an almost impossible to see difference in the character that looks like `o`:

bg-spell-error

@LubenZA it's probably a Google Translate error, but could you please check this? Thanks!

Comment (LubenZA) [reply src/Symfony/Component/Validator/Resources/translations/validators.bg.xlf]: Yes - was wrong charset

**Review by LubenZA — COMMENTED** [review]
Char fix

**Review by javiereguiluz — APPROVED** [review]
Thanks a lot @LubenZA for your contribution!

**Comment by javiereguiluz** [comment]
@LubenZA I'm sorry to bother you, but this morning we added a new phrase to the translated files. Could you please add this new translation so we complete the Bulgarian translation? Thanks!

```xml
<trans-unit id="86">
    <source>This value should be valid JSON.</source>
    <target>...</target>
</trans-unit>
```

**Comment by LubenZA** [comment]
No bother at all - I will translate shortly.

**Review by LubenZA — COMMENTED** [review]
Added 86.

**Comment by LubenZA** [comment]
If you need more translations   just shout.
Feel free to close when you are happy.

**Comment by javiereguiluz** [comment]
@LubenZA thank you! This is now perfectly completed. We'll merge your pull request soon. Thanks.

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @LubenZA.

**Pull request merged**